src/main/menu/menu-registration.d.ts

```diff
@@ -18,15 +18,8 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
```

Comment:
Also, why move this file?

Reply:
This was located in a file which was deleted.

Reply:
Ah well the git diff implied that it was moved here instead. Then why delete that file in registries instead of just deleting the registry?

Reply:
Would be stupid to leave empty file there?  (Or file that doesn’t do anything and doesn’t have anything to do with registries.)

Before it was the place that this interface was used first time, but since that doesn’t exist anymore, it makes sense to move it closer to useplace.

Reply:
It is also used in `LensMainExtension` and I would argue that is its "first use" and this is merely its consumer.

Reply:
I would say that `interface MenuRegistration` exists for making sure that `Menu` has items with proper interface. That being said, `Menu` is the one that defines the requirements for the interface -> logical place for the interface is next to `Menu`.